cmd/auth.go

```diff
@@ -88,22 +94,29 @@ func (d *dexterOIDC) createOauth2Config() error {
 	}
 
 	// setup oidc client context
-	ctx := oidc.ClientContext(context.Background(), d.httpClient)
-
-	// initialize oidc provider
-	provider, err := oidc.NewProvider(ctx, "https://accounts.google.com")
-
-	if err != nil {
-		return errors.New(fmt.Sprintf("failed to create new dexterOIDC provider: %s", err))
-	}
+	oidc.ClientContext(context.Background(), d.httpClient)
 
 	// populate oauth2 config
 	d.Oauth2Config.ClientID = oidcData.clientID
 	d.Oauth2Config.ClientSecret = oidcData.clientSecret
 	d.Oauth2Config.RedirectURL = oidcData.callback
-	d.Oauth2Config.Endpoint = provider.Endpoint()
-	d.Oauth2Config.Scopes = []string{oidc.ScopeOpenID, "profile", "email"}
 
+	switch oidcData.endpoint {
+	case "azure":
+		d.Oauth2Config.Endpoint = microsoft.AzureADEndpoint(oidcData.azureTenant)
+		d.Oauth2Config.Scopes = []string{oidc.ScopeOpenID, oidc.ScopeOfflineAccess, "email"}
+	case "google":
+		d.Oauth2Config.Endpoint = google.Endpoint
+		d.Oauth2Config.Scopes = []string{oidc.ScopeOpenID, "profile", "email"}
+	default:
+		return errors.New(fmt.Sprintf("unsupported endpoint: %s", oidcData.endpoint))
+	}
+		d.Oauth2Config.Endpoint = microsoft.AzureADEndpoint(oidcData.azureTenant)
```

Comment:
Thanks for accepting my code suggestions. Unfortunately github is not super flexible when it comes to multiline/block changes. Line #114 - #119 in the diff would need to be removed as well. Sorry

Reply:
sorry, didn't see them until I pulled locally. Should be fixed now. What do I do to satisfy buildstep 3 in Travis?